assets/sass/components/dashboard/_googlesitekit-ReportMetric.scss

```diff
@@ -18,6 +18,10 @@
 
 .googlesitekit-pagespeed-report-metric {
 
```

Comment:
What do you think of using a `gap: 20px`, or `30px` here on the tr wrapper? It will allow the text to have more space on mobile vs narrowing it down by reducing `100px` in all instances